Makefile

```diff
@@ -32,7 +33,7 @@ output/docs.pdf: $(DOC_FILES)
 	-v $(shell pwd)/:/input/:ro \
 	-v $(shell pwd)/output/:/output/ \
 	-u $(shell id -u) \
-	vbatts/pandoc -f markdown_github -t latex -o /$@ $(patsubst %,/input/%,$(DOC_FILES))
+	$(PANDOC) -f markdown_github -t latex -o /$@ $(patsubst %,/input/%,$(DOC_FILES))
```

Comment:
On Thu, May 19, 2016 at 08:40:21PM -0700, Lai Jiangshan wrote:
> > -	vbatts/pandoc -f markdown_github -t latex -o /$@ $(patsubst %,/input/%,$(DOC_FILES))
> > +	$(PANDOC) -f markdown_github -t latex -o /$@ $(patsubst %,/input/%,$(DOC_FILES))
> 
> I'm just curious before any test or investigation, $(PANDOC) is a
> **path to the pandoc executive** when default, but we expect
> **pandoc image** here, how this replacement works?

Oops, I had missed this.  See #440 for some paths forward.